### PR TITLE
WF-63706-How to set the background for master slide in the PowerPoint Presentation

### PR DESCRIPTION
**Pull request by Suriya-Balamurugan**
Kindly review the below merge request and let me know your feedback.

<table>
<tr>
<td>
Task Link
</td>
<td>
<a href="https://syncfusion.atlassian.net/browse/WF-63706">https://syncfusion.atlassian.net/browse/WF-63706</a>
</td>
</tr>
<tr>
<td>
Description
</td>
<td>
Add UG document to set the background for master slide in the PowerPoint Presentation.
</td>
</tr>
<tr>
<td>
Solution
</td>
<td>
Added code snippet to to set the background for master slide in the PowerPoint Presentation in all platforms. 
</td>
</tr>
</table>

### CODE REVIEW

**Review by MohanaselvamJothi — CHANGES_REQUESTED** [review]

File: File-Formats/Presentation/create-read-edit-master-slide-and-layout-slide-in-c-sharp.md [hunk 1]
```diff
@@ -77,6 +77,136 @@ pptxDoc.Close()
 
 {% endtabs %}
 
+## Changing MasterSlide background
```
Comment (MohanaselvamJothi) [inline File-Formats/Presentation/create-read-edit-master-slide-and-layout-slide-in-c-sharp.md:80]: Change background of Master slide

**Review by MohanaselvamJothi — CHANGES_REQUESTED** [review]

File: File-Formats/Presentation/create-read-edit-master-slide-and-layout-slide-in-c-sharp.md [hunk 2]
```diff
@@ -77,6 +77,136 @@ pptxDoc.Close()
...
+//Creates a PowerPoint presentation.
+using(IPresentation pptxDoc = Presentation.Create())
+{
+    //Accesses the first master slide in PowerPoint file.
+    IMasterSlide masterSlide = pptxDoc.Masters[0];
+    //Retrieves the background instance.
+    IBackground background = masterSlide.Background;
+    //Sets the fill type for background as Solid fill.
+    background.Fill.FillType = FillType.Solid;
+    //Creates instance for solid Fill.
```
Comment (Ramaraj-Marimuthu) [inline File-Formats/Presentation/create-read-edit-master-slide-and-layout-slide-in-c-sharp.md:97]: @Suriya-Balamurugan - Please modify the XML comment as `Gets the instance of solid Fill.`

**Review by Ramaraj-Marimuthu — CHANGES_REQUESTED** [review]

File: File-Formats/Presentation/create-read-edit-master-slide-and-layout-slide-in-c-sharp.md [hunk 3]
```diff
@@ -77,6 +77,136 @@ pptxDoc.Close()
 
 {% endtabs %}
 
+## Changing MasterSlide background
+
+The following code example demonstrates how to set the background for a master slide.
```
[the remark this answers is not in the document]
Comment (Suriya-Balamurugan) [reply File-Formats/Presentation/create-read-edit-master-slide-and-layout-slide-in-c-sharp.md]: Addressed

File: File-Formats/Presentation/create-read-edit-master-slide-and-layout-slide-in-c-sharp.md (hunk 2 above)
Comment (Suriya-Balamurugan) [reply File-Formats/Presentation/create-read-edit-master-slide-and-layout-slide-in-c-sharp.md]: Addressed.

File: File-Formats/Presentation/create-read-edit-master-slide-and-layout-slide-in-c-sharp.md (hunk 1 above)
Comment (Suriya-Balamurugan) [reply File-Formats/Presentation/create-read-edit-master-slide-and-layout-slide-in-c-sharp.md]: Addressed.

File: File-Formats/Presentation/create-read-edit-master-slide-and-layout-slide-in-c-sharp.md [hunk 4]
```diff
@@ -1,6 +1,6 @@
 ---
 title: Create, read and edit Master and Layout slides in CSharp |Syncfusion|
-description: Create, read and edit Master slides and Layout slides in CSharp
+description: Create, read and edit Master slides and Layout slides in CSharp using Syncfusion PowerPoint Presentation library.
```
Comment (MohanaselvamJothi) [inline File-Formats/Presentation/create-read-edit-master-slide-and-layout-slide-in-c-sharp.md:3]: Create, read and edit Master slides and Layout slides using Syncfusion PowerPoint library (Essential Presentation).

**Review by MohanaselvamJothi — CHANGES_REQUESTED** [review]

File: File-Formats/Presentation/create-read-edit-master-slide-and-layout-slide-in-c-sharp.md [hunk 5]
```diff
@@ -77,6 +77,136 @@ pptxDoc.Close()
 
 {% endtabs %}
 
+## Change background of Master slide
+
+You can change the background of the master slide, all slides in the presentation would receive the same background settings. The following code example demonstrates how to set the background for a master slide.
```
Comment (MohanaselvamJothi) [inline File-Formats/Presentation/create-read-edit-master-slide-and-layout-slide-in-c-sharp.md:82]: Please get content review

**Review by MohanaselvamJothi — CHANGES_REQUESTED** [review]

File: File-Formats/Presentation/create-read-edit-master-slide-and-layout-slide-in-c-sharp.md (hunk 4 above)
Comment (Suriya-Balamurugan) [reply File-Formats/Presentation/create-read-edit-master-slide-and-layout-slide-in-c-sharp.md]: Addressed

File: File-Formats/Presentation/create-read-edit-master-slide-and-layout-slide-in-c-sharp.md (hunk 5 above)
Comment (Suriya-Balamurugan) [reply File-Formats/Presentation/create-read-edit-master-slide-and-layout-slide-in-c-sharp.md:82]: Content reviewed.

**Review by MohanaselvamJothi — APPROVED** [review]

**Review by MohanaselvamJothi — APPROVED** [review]

**Review by Ramaraj-Marimuthu — APPROVED** [review]
I don't have any concerns in this code changes.

**Pull request merged**